.gitignore

```diff
@@ -20,3 +20,4 @@ lib/
 log/
 target/
 dependency-reduced-pom.xml
+/bookkeeper-server/nbproject/
```

Comment:
what is this?

Reply:
Sorry, will drop. It is a netbeans empty folder

Reply:
Sorry, will drop. It is a netbeans empty folder

Reply:
done